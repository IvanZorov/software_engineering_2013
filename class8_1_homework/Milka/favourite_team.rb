require 'csv'
played = Array.new()
team = ARGV[1]
CSV.foreach(ARGV[0]) do |csv|
	
	played << csv if team == csv[1].split("-")[0] || team == csv[1].split("-")[1]
end
played.sort!
CSV.open("favourite_team_result.csv", "w") do |row|
	played.each do |element|
		row << element
	end
end
