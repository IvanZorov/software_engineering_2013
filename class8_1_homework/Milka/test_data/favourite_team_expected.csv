19/10/2013,LAL-MIA,121-115
20/10/2013,LAC-MIA,67-54
21/10/2013,MIA-CHI,123-133
22/10/2013,MIA-LAC,77-145
23/10/2013,LAL-MIA,76-111
24/10/2013,LAC-MIA,146-126
25/10/2013,MIA-CHI,88-126
26/10/2013,MIA-LAC,123-125
